source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -18,16 +18,51 @@ Do _NOT_ let certificates expire. If they expire, the environment becomes non-re
 . Once the host is in maintenance mode, click menu:Installation[Enroll Certificate].
 . Once enrollment is complete, click menu:Management[Activate].
 
-.Next steps
-You must also renew Engine certificates.
+[IMPORTANT]
+====
+You must also renew {engine-name} certificates.
+====
 
-. Run `engine-setup` from an SSH or console session on the {engine-appliance-name} system. The `engine-setup` script prompts you with some configuration questions.
+.Renewing certificates in a standalone {engine-name}
+. Log in as root via SSH or console session to the {engine-name} and run the following command:
 +
 ----
-# engine-setup
+# engine-setup --offline.
 ----
 +
+The `engine-setup` script prompts you with configuration questions.
+
 . Respond to the questions as appropriate or use an answers file.
++
+Be sure to answer the question about renewing certificates with "Y." This will renew the certificates without applying any updates.
++
+If you want to apply updates, follow the procedure in the Upgrade Guide.
+
+.Renewing certificates in a self-hosted engine
+
+. Log in as root via SSH or console session to the {engine-name} and a self-hosted engine hypervisor.
+
+. In the hypervisor, run the following command to place the environment in global maintenance mode:
++
+----
+# hosted-engine --set-maintenance –mode=global
+----
+. In the {engine-name}, run the following command to renew the certificates:
++
+----
+# engine-setup --offline.
+----
+
+. Respond to the questions as appropriate or use an answers file.
+Be sure to answer the question about renewing certificates with "Y." This will renew the certificates without applying any updates.
++
+If you want to apply updates, follow the procedure in the Upgrade Guide.
+
+. In the hypervisor, run the following command to remove the environment from global maintenance mode:
++
+----
+# hosted-engine --set-maintenance –mode=none
```

Comment:
Change last option to `--mode=none`.

Reply:
looks like auto correct in my draft document turned the two hyphens into a long one

Reply:
I was wondering about that. I didn't think it was possible to use an en-dash in Atom! :-)